tests/test_freshness_date_parser.py

```diff
@@ -107,6 +107,7 @@ def setUp(self):
         param('anteontem', ago={'days': 2}, period='day'),
         param('hoje', ago={'days': 0}, period='day'),
         param('uma hora atrás', ago={'hours': 1}, period='day'),
+        param('1 segundo atrás', ago={'seconds': 1}, period='day'),
```

Comment:
Should there also be another check in test_translation under the Portuguese section?

```python
param('pt', "1 segundo atrás", "1 second ago"),
```